docs/src/content/en/docs/server-db/custom-api-routes.mdx

```diff
@@ -25,31 +23,45 @@ export const mastra = new Mastra({
           const mastra = c.get("mastra");
           const agents = await mastra.getAgent("my-agent");
 
-          return c.json({ message: "Hello, world!" });
+          return c.json({ message: "Custom route" });
         },
       }),
     ],
   },
 });
 ```
 
-Each route's handler receives the Hono `Context`. Within the handler you can
-access the `Mastra` instance to fetch or call agents and workflows.
-
-To add route-specific middleware pass a `middleware` array when calling
-`registerApiRoute`.
-
-```typescript copy showLineNumbers
-registerApiRoute("/my-custom-route", {
-  method: "GET",
-  middleware: [
-    async (c, next) => {
-      console.log(`${c.req.method} ${c.req.url}`);
-      await next();
-    },
-  ],
-  handler: async (c) => {
-    return c.json({ message: "My route with custom middleware" });
-  },
+Once registered, a custom route will be accessible from the root of the server. For example:
+
+```bash
+curl http://localhost:4111/my-custom-route
```

Comment:
@wardpeet / @abhiaiyer91  Added this to explain where custom route are accessible. Does this address the issue raised in: https://github.com/mastra-ai/mastra/issues/6090